libs/register/src/lib/+state/register.state.ts

```diff
@@ -0,0 +1,94 @@
+//implements F010
+import { Injectable } from '@angular/core';
+import { ComponentStore, tapResponse } from '@ngrx/component-store';
+import { AccountDTO, AccountRegisterDTO, ApiError } from '@party-time/models';
+import { RegisterService } from '../services/register.service';
+import { tap, exhaustMap, Observable } from 'rxjs';
+
+export interface RegisterStateInterface {
+  isRegisterd?: boolean; // has the Register Api request been completed
+  accountRegisterDTO?: AccountRegisterDTO; // last known value (if any)
+  accountDTO?: AccountDTO; // last known value (if any)
+  loading: boolean; // is this request loading
+  error?: ApiError | null; // last known error (if any)
+}
+
+export const initialRegisterState: RegisterStateInterface = {
+  isRegisterd: false,
+  loading: false,
+  accountRegisterDTO: undefined,
+  accountDTO: undefined,
+  error: null,
+};
+
+@Injectable()
+export class RegisterStore extends ComponentStore<RegisterStateInterface> {
+  private isLoading$ = this.select((state) => state.loading);
+  private isRegisterd$ = this.select((state) => state.isRegisterd);
```

Comment:
```suggestion
  private isRegistered$ = this.select((state) => state.isRegistered);
```

@JulianTurner Seiteneffekte dieser Änderung?